backend/onyx/configs/app_configs.py

```diff
@@ -179,6 +179,25 @@
 REDIS_PORT = int(os.environ.get("REDIS_PORT", 6379))
 REDIS_PASSWORD = os.environ.get("REDIS_PASSWORD") or ""
 
+# Rate limiting for auth endpoints
+
+
+RATE_LIMIT_WINDOW_SECONDS: int | None = None
```

Comment:
Could be a shared util function to do something like
def _safe_int_env(env_var: str) -> int | None: